Sources/_CryptoExtras/RSA/RSA_security.swift

```diff
@@ -43,6 +43,10 @@ internal struct SecurityRSAPublicKey: @unchecked Sendable {
         self.backing = unwrappedKey
     }
 
+    init(_ boringSSLKey: BoringSSLRSAPublicKey) throws {
+        try self.init(pemRepresentation: boringSSLKey.pemRepresentation)
```

Comment:
It's probably a touch cheaper to use the DER representation.

Reply:
Thanks, done!